rosidl_generator_cpp/include/rosidl_generator_cpp/bounded_vector.hpp

```diff
@@ -54,7 +54,7 @@ class BoundedVector
 
   /// Create a %BoundedVector with no elements.
   BoundedVector()
-  noexcept(std::is_nothrow_default_constructible<_Alloc>::value)
+  noexcept (std::is_nothrow_default_constructible<_Alloc>::value)
```

Comment:
I don't think this is right, since it's actually calling `noexcept(...)`: http://en.cppreference.com/w/cpp/language/noexcept

Reply:
This is what it takes to comply with `uncrustify`. Please feel free to fill a ticket upstream. (I just don't think it is worth it.)

Reply:
I anyway took the time to fill a ticket upstream: uncrustify/uncrustify#1649

I don't think we should be waiting for that though.

Reply:
Sounds good.